### PR TITLE
301/add specs

### PR DESCRIPTION
**Pull request by ebrett**
### Context
Just adding some more specs

### CODE REVIEW

File: spec/features/admin/user_admin_page.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.feature "User administration", type: :feature do
+  before do
+    create :admin, first_name: "Barbara", last_name: "Gordon", email: "birdsofprey@oracle.com"
+    create :editor, first_name: "Perry", last_name: "White", email: "pwhite@dailyplanet.com"
```
Comment (guy-roberts) [inline spec/features/admin/user_admin_page.rb:6]: You can use  

FactoryBot.create :user, role: 'admin'
FactoryBot.create :user, role: 'editor'

or add a couple of traits to the factory

Comment (guy-roberts) [reply spec/features/admin/user_admin_page.rb:6]: That way, if the model gets a new attribute, you only need to update the factory in one place

Comment (guy-roberts) [reply spec/features/admin/user_admin_page.rb:6]: A bit like the way parent_page and child_page are done in 

https://github.com/DFE-Digital/early-years-foundation-reform/blob/main/spec/features/content_pages_spec.rb

**Review by guy-roberts — APPROVED** [review]

File: spec/features/admin/user_admin_page.rb (hunk 1 above)
Comment (ebrett) [reply spec/features/admin/user_admin_page.rb:6]: I am using factory bot - and rather than traits I'm using aliases.  I think it works better.

Comment (guy-roberts) [reply spec/features/admin/user_admin_page.rb:6]: Oh ok

**Pull request merged**